packages/js/cli/src/__tests__/commands.spec.ts

```diff
@@ -227,6 +227,7 @@ const testData: CommandTestCaseData<CommandTypings> = {
     init: [{
       cwd: fs.mkdtempSync(path.join(os.tmpdir(), "docs-init")),
       before: async (test) => {
+        fs.writeFileSync(`${test.cwd}/polywrap.yaml`,"");
```

Comment:
`polywrap docs init` doesn't create polywrap manifest so we need to have polywrap manifest to create docs manifest